### ISSUE TITLE
Add test vector for EIP-1271 contract resolution

### ISSUE THREAD

**Issue by w4ll3** [spruceid/siwe#64]
Currently there are no tests that check if the checkContractWalletAddress is working accordingly.

**Comment by awoie** [comment]
@w4ll3 Please check whether this issue has been fixed already.

**Comment by w4ll3** [comment]
This wasn't solved yet because tests would fail in languages that does not
support that, should we make an exclusive test for typescript only?

**Comment by awoie** [comment]
@w4ll3 yes, i think this would be a good idea. we won't have always feature parity between siwe core packages. at least not always at the same time.

**Comment by w4ll3** [comment]
#69
